src/detect-flow-pkts.c

```diff
@@ -23,114 +23,103 @@
 #include "detect-engine-uint.h"
 #include "detect-parse.h"
 
-static int DetectFlowPktsToClientMatch(
+#define DETECT_FLOW_TO_SERVER 1
+#define DETECT_FLOW_TO_CLIENT 2
+#define DETECT_FLOW_TO_EITHER 3
```

Comment:
What about "both" ?

Reply:
And addition of both directions ?

Reply:
> What about "both" ?

Hmm... We can use the two existing keywords to have this `both` behavior already

Reply:
yep

Reply:
And I guess we can have lua for more complex stuff like bytes_to_client > 10 * bytes_to_server + 1000

Reply:
makes sense.

Reply:
So, I guess in the end we just miss `either`